packages/common/src/web3-toolkit.test.ts

```diff
@@ -93,7 +96,13 @@ describe('Web3Toolkit', function () {
       it(`case ${i + 1}`, async function () {
         const web3Toolkit = new Web3Toolkit(chainId);
         const tokens = await web3Toolkit.getTokens(tokenAddresses);
-        expect(JSON.stringify(tokens)).to.eq(JSON.stringify(expected));
+        for (let i = 0; i < tokens.length; i++) {
+          expect(tokens[i].chainId).to.eq(expected[i].chainId);
+          expect(tokens[i].address).to.eq(expected[i].address);
+          expect(tokens[i].decimals).to.eq(expected[i].decimals);
+          expect(tokens[i].symbol).to.eq(expected[i].symbol);
+          expect(tokens[i].name).to.eq(expected[i].name);
```

Comment:
Can we keep to use `JSON.stringify` but modify the above `expected`?

Reply:
fixed with `JSON.stringify`